nf_core/components/update.py

```diff
@@ -868,7 +878,17 @@ def get_components_to_update(self, component):
         if self.component_type == "modules":
             # All subworkflow names in the installed_by section of a module are subworkflows using this module
             # We need to update them too
-            subworkflows_to_update = [subworkflow for subworkflow in installed_by if subworkflow != self.component_type]
+            git_remote = self.current_remote
+            for subworkflow in installed_by:
+                if subworkflow != component:
+                    for remote_url, content in mods_json["repos"].items():
+                        if (all_subworkflows := content.get("subworkflows")) is not None:
```

Comment:
Can `content` contain false-ish values that are not None ? Otherwise this may suffice (I admit it's a matter of taste)
```suggestion
                        if all_subworkflows := content.get("subworkflows"):
```